src/ExamplePairings/ExamplePairings.js

```diff
@@ -4,7 +4,7 @@ import { Typography, List, ListItem, ListItemText, Grid, MenuItem, Menu, withSty
 
 const styles = theme => ({
   header: {
-    marginLeft: theme.spacing.unit * 3,
+   
```

Comment:
are these margin changes specific to error handling? otherwise thinking they should have their own commit

Reply:
these changes make the errors / params / result headings all align visually, so I think it should be OK getting in in this PR